composer.json

```diff
@@ -15,22 +15,17 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
-        "contao/core-bundle": "4.4.*",
-        "symfony/framework-bundle": "^3.3"
-    },
-    "conflict": {
-        "contao/core": "*",
-        "contao/manager-plugin": "<2.0 || >=3.0"
```

Comment:
I have removed the conflicts as I am not sure what the purpose of these conflicts would be. But may be someone can enlighten me ;)

Reply:
/cc @aschempp

Reply:
Ah I think for the `contao/manager-plugin` it's a way of requiring `^2.0`, just for the Managed Edition, without creating a hard dependency.

But the `contao/core` one I still don't really get, since the `contao/core-bundle` requirement already prevents it from being installed in a Contao 3 instance?